### ISSUE TITLE
Forward loss calculate in the NodePropertyPrediction

### ISSUE THREAD

**Issue by fourier20laplace** [DeepGraphLearning/torchdrug#256]
![image](https://github.com/user-attachments/assets/e6f3dd7c-5257-401a-9503-a680115b5ec8)
why there is a " all_loss += loss " in the end , I think it should have been done by the weighted process of "all_loss += loss *weight" 

Thank you for your attention to this matter, and I look forward to hearing from you.
